src/controllers/aer_controller.hpp

```diff
@@ -887,20 +949,61 @@ Result Controller::execute(std::vector<Circuit> &circuits,
 #endif
     }
 #endif
+
     // then- and else-blocks have intentionally duplication.
     // Nested omp has significant overheads even though a guard condition
     // exists.
     const int NUM_RESULTS = result.results.size();
-    if (parallel_experiments_ > 1) {
-#pragma omp parallel for num_threads(parallel_experiments_)
+    if (parallel_experiments_ > 1 && sim_device_ != Device::GPU && !multi_chunk_req) {
```

Comment:
can we set `parallel_experiments = 1` in the extended `set_parallelization_experiments()`? Then this change is not necessary.